scipy/sparse/linalg/eigen/arpack/tests/test_arpack.py

```diff
@@ -26,21 +24,6 @@
 from scipy.linalg import svd, hilbert
 
 from scipy._lib._gcutils import assert_deallocated
-from scipy._lib._numpy_compat import suppress_warnings
-
-
-# eigs() and eigsh() are called many times, so apply a filter for the warnings
-# they generate here.
-sup = suppress_warnings()
-sup.filter(message="Single-precision types in `eigs` and `eighs`")
```

Comment:
I found something about the single precision version of these functions being removed in 1.10 in the logs or so, I assume these are simply not needed anymore.